### PR TITLE
CLI import: add skip support to bulk files

### PR DESCRIPTION
**Pull request by sbesson**
Follow-up of #5848, this PR adds support for `skip` option in bulk import configuration files. The various YAML files under ` components/tools/OmeroPy/test/unit/clitest/bulk_import/test_skip` should test all the values currently supported by the import plugin. The command-line usage of `--skip` option should be unmodified by this PR so integration tests are expected to pass.

### CODE REVIEW

File: components/tools/OmeroPy/test/unit/clitest/bulk_import/test_skip/upgrade.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+skip: "all"
```
Comment (dominikl) [inline components/tools/OmeroPy/test/unit/clitest/bulk_import/test_skip/upgrade.yml]: Shouldn't this be `upgrade`?

**Comment by dominikl** [comment]
Would the integration test better fit into `test_import_bulk.py` instead of `test_import.py`?

**Comment by joshmoore** [comment]
:+1: from me for splitting the test files into smaller portions moving forward.

**Comment by dominikl** [comment]
Apart from that, looks good. Also tested manually, works fine 👍

File: components/tools/OmeroPy/test/unit/clitest/bulk_import/test_skip/upgrade.yml (hunk 1 above)
Comment (sbesson) [reply components/tools/OmeroPy/test/unit/clitest/bulk_import/test_skip/upgrade.yml]: You're right. Thanks for spotting that

**Comment by sbesson** [comment]
I'll push a commit to address https://github.com/openmicroscopy/openmicroscopy/pull/5850#discussion_r216266818. Splitting bulk tests from `test_import.py` into `test_import_bulk.py` make sense and mirror the integration tests. Happy to also do that as part in this PR although my main concern is to conflict with #5849.

**Comment by joshmoore** [comment]
Happy for us to go ahead and merge #5849 if that helps. I tried to put the readers test far away from the bulk tests for just this reason.

**Comment by joshmoore** [comment]
Merging based on travis status. Happy to have test file reworking happen down the road.

**Pull request merged**